### ISSUE TITLE
xpcs-viewer shows 'damaged files, Matlab GUI works OK

### ISSUE THREAD

**Issue by qzhang234** [AdvancedPhotonSource/pyXpcsViewer#2]
@AZjk  Here's the error message on `xpcs-viewer`. It works fine from the Matlab GUI:

<details>
<summary> Terminal output </summary>

```

2021-03-12 18:06:29,174 xpcs_viewer.module.average_toolbox: create file: /home/8ididata/2021-1/lurio202103/cluster_results/AvgE017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00001_0001-100000.hdf
2021-03-12 18:06:29,209 xpcs_viewer.module.average_toolbox: average job 0 finished
2021-03-12 18:06:44,959 xpcs_viewer.file_locator: length of target = 46
2021-03-12 18:06:44,969 xpcs_viewer.viewer      : use the previous save path
2021-03-12 18:06:52,703 xpcs_viewer.viewer_kernel: create average job, ID = 1
2021-03-12 18:06:53,710 xpcs_viewer.viewer      : disconnect previous slot
2021-03-12 18:06:56,111 xpcs_viewer.module.average_toolbox: average job 1 starts
2021-03-12 18:06:56,135 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00001_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,151 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00002_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,160 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00003_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,168 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00004_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,175 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00005_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,182 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00006_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,189 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00007_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,196 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00008_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,202 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00009_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,208 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00010_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,214 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00011_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,220 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00012_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,227 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00013_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,233 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00014_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,239 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00015_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,245 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00016_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,250 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00017_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,256 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00018_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,262 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00019_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,268 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00020_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,274 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00021_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,280 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00022_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,286 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00023_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,292 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00024_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,299 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00025_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,305 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00026_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,311 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00027_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,317 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00028_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,323 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00029_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,330 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00030_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,336 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00031_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,341 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00032_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,347 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00033_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,353 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00034_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,359 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00035_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,365 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00036_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,371 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00037_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,377 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00038_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,383 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00039_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,389 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00040_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,396 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00041_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,402 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00042_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,410 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00043_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,416 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00044_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,423 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00045_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,429 xpcs_viewer.module.average_toolbox: file E017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00046_0001-100000.hdf is damaged, skip
2021-03-12 18:06:56,429 xpcs_viewer.module.average_toolbox: create file: /home/8ididata/2021-1/lurio202103/cluster_results/AvgE017_alpha4gamma1_286_Fly1250umpersec_att0_Rq1_00001_0001-100000.hdf
2021-03-12 18:06:56,463 xpcs_viewer.module.average_toolbox: average job 1 finished



```

</details>

**Comment by qzhang234** [comment]
I installed the most recent version as of today (not sure if the push has went through). This time the average completed and the averaged file was saved, however I can't open the averaged file and plot the SAXS_2D and g2 in the GUI.

I have attached three of those single-ROI results in case you want to test the GUI behaviors. The files can also be found at:
`/home/8ididata/2021-3/lurio202103/cluster_results/E017_*`

Thanks,

[E017.zip](https://github.com/AdvancedPhotonSource/pyXpcsViewer/files/6135353/E017.zip)
